source/common/common/scope_tracker.h

```diff
@@ -31,7 +36,14 @@ class ScopeTrackerScopeState {
   void* operator new(std::size_t) = delete;
 
 private:
+  friend class ScopeTrackerScopeStateTest;
+  static bool& executionContextEnabled() {
+    static bool enabled =
+        Runtime::runtimeFeatureEnabled("envoy.restart_features.enable_execution_context");
+    return enabled;
+  }
```

Comment:
`static bool executionContextEnabled() { ... }`

Reference semantics make no sense here and may has unexpected side effect.

Reply:
ACK.